recipe/build-lib.bat

```diff
@@ -49,11 +49,12 @@ cmake -B _build ^
     -DCMAKE_INSTALL_LIBDIR="%LIBRARY_LIB%" ^
     -DCMAKE_INSTALL_INCLUDEDIR="%LIBRARY_INC%" ^
     !CUDA_CONFIG_ARGS! ^
+    /verbosity:detailed
```

Comment:
I don't think you can use it here, and we don't need it either! `/verbosity` is only meant to reach MSBuild, not CMake itself, I think.